js/Extensions/Annotations/Popup.js

```diff
@@ -726,6 +737,7 @@ addEvent(NavigationBindings, 'showPopup', function (config) {
             (this.chart.options.stockTools &&
                 this.chart.options.stockTools.gui.iconsURL) ||
             'https://code.highcharts.com/@product.version@/gfx/stock-icons/'));
+        this.popup.chart = this.chart;
```

Comment:
Instead of assigning it here, should we wait for #15015 to be merged? `popup.chart` is already implemented there. 

PS: It seems #15015 is missing `popup.chart` definition in TS Popup class.

Reply:
I will wait till the merge and then I'm going to change that :)